static/app/views/detectors/layouts/new.tsx

```diff
@@ -1,22 +0,0 @@
-import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
```

Comment:
Curious why we're removing this component? it seems like a helpful wrapper component to build more monitors quickly.

Reply:
The layouts were preemptively split out a bit early. Right now they're getting in the way since they're trying to share code between multiple pages that are changing. We'll likely have a single page for edit/new and another for step 1 similar to metric alerts. Right now they don't have a ton in common.